gems/sorbet-runtime/lib/types/props/private/deserializer_generator.rb

```diff
@@ -57,15 +57,11 @@ def self.generate(props, defaults)
               begin
                 #{transformation}
               rescue NoMethodError => e
-                T::Configuration.soft_assert_handler(
-                  'Deserialization error (probably unexpected stored type)',
-                  storytime: {
-                    klass: self.class,
-                    prop: #{prop.inspect},
-                    value: val,
-                    error: e.message,
-                    notify: 'djudd'
-                  }
+                T::Configuration.deserialization_error_handler(
+                  self.class,
+                  #{prop.inspect},
+                  val,
+                  e,
```

Comment:
If the goal is to have something that it easy to call instead of having to reproduce the arguments given to soft_assert_handler, would we not want to just wrap this code into a private helper function? Why add this `T::Configuration`, effectively making it a public API?